<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsWindow</class>
 <widget class="QDialog" name="TagsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="tagsWinList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tagsWinDoneButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="tagsWinRemoveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Remove Tag</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="tagsWinAddButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Add Tag</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tagsWinAddLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name of New Tag</string>
   </property>
  </widget>
  <widget class="QLabel" name="tagsWinMessageLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>305</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tagsWinList</zorder>
  <zorder>tagsWinRemoveButton</zorder>
  <zorder>tagsWinAddButton</zorder>
  <zorder>tagsWinAddLineEdit</zorder>
  <zorder>tagsWinDoneButton</zorder>
  <zorder>tagsWinMessageLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
